src/app/components/ScrollablePromo/PromoList/index.jsx

```diff
@@ -77,21 +77,25 @@ const OperaStyledList = styled.li`
       margin-${dir === 'ltr' ? `left` : `right`}: 0;}`}
 `;
 
-const PromoList = ({ blocks }) => {
+const PromoList = ({ blocks, viewTracker, onClick }) => {
```

Comment:
Is the click tracking going to work at block level? This looks like it's using its own form of a promo, instead of [StoryPromo](https://github.com/bbc/simorgh/blob/latest/src/app/containers/StoryPromo/index.jsx) in containers, and for block-level click tracking to work on other components with StoryPromo, I think it uses the [`useCombinedClickTrackerHandler`](https://github.com/bbc/simorgh/blob/latest/src/app/containers/StoryPromo/useCombinedClickTrackerHandler.js) to make it work, with its usage [here](https://github.com/bbc/simorgh/blob/d823b26c64e976a12da3a5fbd3881544cbfe6db1/src/app/containers/StoryPromo/index.jsx#L138). I'm not entirely sure if this is the case, but I had to make changes on that file for Optimizely click tracking work

Reply:
Yup, it does work on block level. `[useCombinedClickTrackerHandler](https://github.com/bbc/simorgh/blob/latest/src/app/containers/StoryPromo/useCombinedClickTrackerHandler.js)` looks like it is still using `useClickTrackerHandler` so not sure why this function was created and how it differs. We've implemented it to how [bullet list](https://github.com/bbc/simorgh/blob/d823b26c64e976a12da3a5fbd3881544cbfe6db1/src/app/containers/BulletedList/index.jsx#L34) was done. Still seems to track on block level so I don't think we need to use the combined click tracker.